projects/angular-showcase/src/app/business/business-examples/file-selector-showcase/file-selector-showcase.component.html

```diff
@@ -0,0 +1,57 @@
+<article class="container-fluid">
```

Comment:
If you like, split up the examples in 3 small ones, and if you (still ;-) like, do the same in public repo.